src/acl/ConnMark.h

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (C) 1996-2017 The Squid Software Foundation and contributors
+ *
+ * Squid software is distributed under GPLv2+ license and includes
+ * contributions from numerous individuals and organizations.
+ * Please see the COPYING and CONTRIBUTORS files for details.
+ */
+
+
```

Comment:
extra line still present, but I'm going to run the formatter on this to check for tab issues anyway so no need to hold the merge.